release-notes/2.1/README.md

```diff
@@ -1,40 +1,44 @@
 # .NET Core 2.1 Releases
 
-Important: .NET Core 2.1 is scheduled to go out of support on August 21, 2021. You should consider moving your usage to a later release.
+> Important: [.NET Core 2.1 is scheduled to go out of support on August 21, 2021](https://devblogs.microsoft.com/dotnet/net-core-2-1-will-reach-end-of-support-on-august-21-2021/). You should consider moving your applications to a later release.
 
-[.NET Core 2.1](https://blogs.msdn.microsoft.com/dotnet/2018/05/30/announcing-net-core-2-1) was released on May 30th, 2018. The release was a collaborative effort by many [contributors](2.1.0-contributor.md). .NET Core 2.1 is [supported by Microsoft](../../microsoft-support.md) on [multiple operating systems](2.1-supported-os.md).
+[.NET Core 2.1](https://blogs.msdn.microsoft.com/dotnet/2018/05/30/announcing-net-core-2-1) is an [LTS release](../../release-policies.md) and will be supported for (a little more than) three years, from May 2018 to August 2021. It is [supported by Microsoft](../../microsoft-support.md) on [multiple operating systems](2.1-supported-os.md).
 
-Release notes and download information is provided for each update, in the follow table. You must be on the latest update to be supported.
+- [Downloads](https://dotnet.microsoft.com/download/dotnet/2.1) (also see release notes)
```

Comment:
What do you think about making `also see release notes` a link to the `Releases` section?

Reply:
Good call.